src/platform_linux/platform.c

```diff
@@ -82,12 +82,13 @@ platform_buffer_init(buffer_handle *bh, size_t length)
    }
 
    if (platform_use_mlock) {
-      int rc = mlock(bh->addr, length);
-      if (rc != 0) {
+      int mlock_rv = mlock(bh->addr, length);
```

Comment:
Rename existing `rc` which shadows the  new instance added at the top of this function (in an earlier commit of this PR cycle).